src/nerve/core/nodes/history.py

```
@@ -479,14 +487,17 @@ def create(
         if base_dir is None:
             base_dir = Path.cwd() / ".nerve" / "history"
 
-        file_path = base_dir / server_name / f"{node_id}.jsonl"
+        file_path = base_dir / server_name / session_name / f"{node_id}.jsonl"
 
         if not file_path.exists():
-            raise FileNotFoundError(f"No history for node '{node_id}' on server '{server_name}'")
+            raise FileNotFoundError(
+                f"No history for node '{node_id}' in session '{session_name}' on server '{server_name}'"
+            )
```

Ruff (0.14.10)

Avoid specifying long messages outside the exception class

(TRY003)